bigquery_etl/cli/query.py

```diff
@@ -581,15 +585,22 @@ def backfill(
 
     for query_file in query_files:
         query_file_path = Path(query_file)
-        metadata = Metadata.of_query_file(str(query_file_path))
-        depends_on_past = metadata.scheduling.get("depends_on_past", False)
+
+        try:
+            metadata = Metadata.of_query_file(str(query_file_path))
+            depends_on_past = metadata.scheduling.get("depends_on_past", False)
+        except FileNotFoundError:
+            depends_on_past = False
 
         if depends_on_past and exclude != []:
             click.echo(
                 f"Warning: depends_on_past = True for {query_file_path} but the"
                 f"following dates will be excluded from the backfill: {exclude}"
             )
 
+        if init:
+            ctx.invoke(initialize, name=query_file, dry_run=dry_run)
```

Comment:
Maybe instead of having an `init` flag, we can check for existence of the table via dry run. If the table does not exist, then we run `initialize`.

For the Airflow task, we need to have some a single invocation that runs successfully for both the initialization and the incremental update case. That could be `bqetl query backfill --init` but in that case we need to make sure that the init is a no-op if the table already exists.

Reply:
Instead of doing a dry run, it's probably easier to instantiate a BigQuery client and check if the table exists. When running the backfill BigQuery access is required anyway

Reply:
It looks like we now rely on the existence check and ignore the `init` argument. Should we remove `init`?

Reply:
Ah, yes. Removed